static/js/publisher/metrics/graphs/activeDevices.js

```diff
@@ -210,7 +210,16 @@ function drawGraph(holderSelector, holder, activeDevices) {
 
   // Add the x axix
   let tickValues = [];
-  if (isMobile() && _data.length > 90) {
+  let tickFormat = "%b %e";
+
+  if (_data.length > 360) {
```

Comment:
Could this part get a bit of comment explaining what is being changed in these conditions?
Seems like formatting changes based on size of data we have? Why is something different happening on mobile?

Reply:
:+1: I also want to add a comment to the python bit too